mindmaps-engine/src/test/java/io/mindmaps/engine/backgroundtasks/InMemoryTaskManagerTest.java

```diff
@@ -93,14 +95,13 @@ public void testResume() {
     }
 
     @Test
-    public void testRestart() throws Exception {
+    public void testRestart() {
```

Comment:
Is this test reliable ? What is stopping takes longer than anticipated ? `Thread.sleeps()` can be annoying but now I am not sure if this test won't be subject to timing issues.  Is the status changing to `STOPPED` instantaneous ?